### ISSUE TITLE
Composing with localClassNameBindings fails

### ISSUE THREAD

**Issue by max** [salsify/ember-css-modules#48]
I have [a sample repo](https://github.com/max/ember-css-modules-bug-compose/commit/9277a3813f4d61a1dc2ed0066ee4a94094b15aa3) here where I try to compose a CSS class from another component and then apply said class conditionally with `localClassNameBindings`. This results in the error:

```
Error: Assertion Failed: classNameBindings must not have spaces in them. Multiple class name bindings can be provided as elements of an array, e.g. ['foo', ':bar']
```

**Comment by dfreeman** [comment]
Thanks for the detailed reproduction! I'll take a look later on today — I suspect I know what's going on, and it should be pretty straightforward to fix.

**Comment by dfreeman** [comment]
This should be fixed in 0.5.0
